app/cli/main.py

```
@@ -97,7 +96,7 @@ def output_hgar(output_dir: str, prefix: str = None):
                     count += 1
         else:
             # 导出所有 HAR，按目录结构（逐个处理避免内存占用）
-            print("Exporting all HAR files with original directory structure")
+            print(f"Exporting all HAR files with original directory structure")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)